specification.txt

```diff
@@ -1549,28 +1549,30 @@ Citation-specific Options
 Disambiguation
 ''''''''''''''
 
-A cite is ambiguous when it matches multiple bibliographic entries [#]_. There
-are four methods available to eliminate such ambiguity:
+A cite is ambiguous when it matches multiple bibliographic entries [#]_. Four
+methods are available to eliminate such ambiguity, which are always tried in the
+following order:
 
 1. Show more names
 2. Expand names (adding initials or full given names)
```

Comment:
```suggestion
1. Expand names (adding initials or full given names)
2. Show more names
```

If expand names is used, that should take place before adding names, no? Otherwise, this paragraph never activates:

<blockquote>
    If cites cannot be (fully) disambiguated by expanding the rendered names,
    and if ``disambiguate-add-names`` is set to "true", then the names still
    hidden as a result of et-al abbreviation after the disambiguation attempt of
    ``disambiguate-add-names`` are added one by one to all members of a set of
    ambiguous cites, until no more cites in the set can be disambiguated by
    adding expanded names.
</blockquote>